tests/queries/0_stateless/01602_array_aggregation.sql

```diff
@@ -3,6 +3,12 @@ SELECT 'Array max ', (arrayMax(array(1,2,3,4,5,6)));
 SELECT 'Array sum ', (arraySum(array(1,2,3,4,5,6)));
 SELECT 'Array avg ', (arrayAvg(array(1,2,3,4,5,6)));
 
+SELECT 'Array min :';
```

Comment:
For the future: This is not ok. First it should have added a new file, and second it should have added way more tests. Adding just the simplest of cases lead to untested code, and as it happened, to multiple bugs being introduced.